src/Chapter3.hs

```diff
@@ -752,6 +843,31 @@ parametrise data types in places where values can be of any general type.
 🕯 HINT: 'Maybe' that some standard types we mentioned above are useful for
   maybe-treasure ;)
 -}
+data Power
+	= Invisibility
+	| Berserk
+	| Freezing -- quite weird thing for a dragon
```

Comment:
Very creative powers! :art: 

And I think `Freezing` is an okay power for a dragon :snowflake: Maybe it's an Ice Dragon or Undead Dragon? :dragon: :dragon_face: